flink-connectors/flink-sql-connector-hive-1.2.2/src/main/resources/META-INF/NOTICE

```diff
@@ -8,6 +8,41 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - org.apache.hive:hive-exec:1.2.2
 - org.apache.hive:hive-metastore:1.2.2
+- org.apache.hive:hive-common:1.2.2
```

Comment:
please separate these entries; one section for ones we bundle directly, another section by what is bundled by hive-exec. This otherwise becomes impossible to verify in the next release; similar to what happened in FLINK-18330.